doc/scapy/layers/automotive.rst

```diff
@@ -913,47 +913,47 @@ Usage example:
 .. image:: ../graphics/animations/animation-scapy-gmlan.svg
 
 
-ECU Utility examples
+Ecu Utility examples
 ====================
 
-The ECU utility can be used to analyze the internal states of an ECU under investigation.
+The Ecu utility can be used to analyze the internal states of an Ecu under investigation.
 This utility depends heavily on the support of the used protocol. ``UDS`` is supported.
 
-Log all commands applied to an ECU
+Log all commands applied to an Ecu
```

Comment:
When using abbreviations, it usually makes more sense to keep it all capitalized. Why are you changing this?

Reply:
I treat `Ecu` as an utility object. I don't wanted to have it capitalized to be easier distinguishable from Layers

Reply:
That makes sense. It might take a bit of time to review the whole PR 😄

Reply:
Yes... I'm really sorry. It started small... then it grew up.